src/EventStore.Core/ClusterVNode.cs

```diff
@@ -526,6 +532,26 @@ public class ClusterVNode :
 					Db.Config.WriterCheckpoint.AsReadOnly(),
 					optimizeReadSideCache: Db.Config.OptimizeReadSideCache));
 
+			var streamInfoCacheCapacity = options.Cluster.StreamInfoCacheCapacity;
+			//qq thoughts
+			// 1. maybe now the default should be 0 (or even -1 to indicate dynamic or something)
+			// 2. i wonder if we should base the calculation off of total system memory rather than free memory, not sure.
```

Comment:
We should stick with free memory just in case something else is running alongside Event Store.